src/Common/MoistThermodynamics/MoistThermodynamics.jl

```diff
@@ -737,6 +738,49 @@ PhasePartition(ts::PhaseDry{FT}) where {FT<:Real} = PhasePartition(FT(0), FT(0),
 PhasePartition(ts::PhaseEquil) = PhasePartition_equil(air_temperature(ts), air_density(ts), ts.q_tot)
 PhasePartition(ts::PhaseNonEquil) = ts.q
 
+function roots_∂e_int_∂T(T::FT, e_int::FT, ρ::FT, q_tot::FT) where {FT<:Real}
```

Comment:
I'd just call this function ``∂e_int_sat_∂T`` (because it is just the temperature derivative of saturation internal energy).